libraries/botbuilder-schema/botbuilder/schema/_models_py3.py

```diff
@@ -287,6 +288,28 @@ def __init__(
         self.semantic_action = semantic_action
         self.caller_id = caller_id
 
+    def create_reply(self, text: str = None, locale: str = None):
+        return Activity(
+            type="message",
+            timestamp=datetime.utcnow(),
+            from_property=ChannelAccount(
+                id=self.recipient.id if self.recipient else None,
+                name=self.recipient.name if self.recipient else None,
+            ),
```

Comment:
```suggestion
            ),
            recipient = ChannelAccount(
                id=self.from_property.id if self.from_property else None,
                name=self.from_property.name if self.from_property else None,
            ),
```

ref: https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Schema/ActivityEx.cs#L170